.gitignore

```diff
@@ -2,6 +2,7 @@ bin/
 /_CPack_Packages
 /doc/doxyxml
 /doc/html
```

Comment:
Maybe that is a better approach.  Why was the html directory ignored in the first place?  It seemed like it might contain generated files (from *.rst) that should not be committed to version control.